HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /B4RC0DE-TM/proxy-list/main/SOCKS5.txt (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0xb0f3ac58>: Failed to establish a new connection: [Errno 101] Network is unreachable'))
HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /B4RC0DE-TM/proxy-list/main/SOCKS4.txt (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0xb0f3ab08>: Failed to establish a new connection: [Errno 101] Network is unreachable'))
HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /almroot/proxylist/master/list.txt (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0xafe58c28>: Failed to establish a new connection: [Errno 101] Network is unreachable'))
HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /hanwayTech/free-proxy-list/main/socks5.txt (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0xb0f431a8>: Failed to establish a new connection: [Errno 101] Network is unreachable'))
HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /hanwayTech/free-proxy-list/main/socks4.txt (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0xb0f431f0>: Failed to establish a new connection: [Errno 101] Network is unreachable'))
HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /aslisk/proxyhttps/main/https.txt (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0xb0f436a0>: Failed to establish a new connection: [Errno 101] Network is unreachable'))
HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /hookzof/socks5_list/master/proxy.txt (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0xb0f43aa8>: Failed to establish a new connection: [Errno 101] Network is unreachable'))
HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /HyperBeats/proxy-list/main/socks4.txt (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0xb0f43028>: Failed to establish a new connection: [Errno 101] Network is unreachable'))
HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /B4RC0DE-TM/proxy-list/main/HTTP.txt (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0xb0f3ab50>: Failed to establish a new connection: [Errno 101] Network is unreachable'))
HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /jetkai/proxy-list/main/online-proxies/txt/proxies-socks4.txt (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0xb0f43ee0>: Failed to establish a new connection: [Errno 101] Network is unreachable'))
HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /HyperBeats/proxy-list/main/socks5.txt (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0xb0f43838>: Failed to establish a new connection: [Errno 101] Network is unreachable'))
HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /clarketm/proxy-list/master/proxy-list-raw.txt (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0xb002f568>: Failed to establish a new connection: [Errno 101] Network is unreachable'))
HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /mmpx12/proxy-list/master/socks4.txt (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0xb0f3adf0>: Failed to establish a new connection: [Errno 101] Network is unreachable'))
HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /jetkai/proxy-list/main/online-proxies/txt/proxies-socks5.txt (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0xb0f3a7f0>: Failed to establish a new connection: [Errno 101] Network is unreachable'))
HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /hanwayTech/free-proxy-list/main/http.txt (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0xafe58868>: Failed to establish a new connection: [Errno 101] Network is unreachable'))
HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /manuGMG/proxy-365/main/SOCKS5.txt (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0xb0f435b0>: Failed to establish a new connection: [Errno 101] Network is unreachable'))
HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /roosterkid/openproxylist/main/SOCKS4_RAW.txt (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0xb0f43598>: Failed to establish a new connection: [Errno 101] Network is unreachable'))
HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /hanwayTech/free-proxy-list/main/https.txt (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0xb002f190>: Failed to establish a new connection: [Errno 101] Network is unreachable'))
HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /mmpx12/proxy-list/master/socks5.txt (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0xafe580e8>: Failed to establish a new connection: [Errno 101] Network is unreachable'))
HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /saschazesiger/Free-Proxies/master/proxies/socks4.txt (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0xafe58d78>: Failed to establish a new connection: [Errno 101] Network is unreachable'))
HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /hendrikbgr/Free-Proxy-Repo/master/proxy_list.txt (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0xb0f438e0>: Failed to establish a new connection: [Errno 101] Network is unreachable'))
HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /roosterkid/openproxylist/main/SOCKS5_RAW.txt (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0xb0f3aa00>: Failed to establish a new connection: [Errno 101] Network is unreachable'))
HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /HyperBeats/proxy-list/main/http.txt (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0xb002fd18>: Failed to establish a new connection: [Errno 101] Network is unreachable'))
HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /ShiftyTR/Proxy-List/master/socks4.txt (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0xb0f431c0>: Failed to establish a new connection: [Errno 101] Network is unreachable'))
HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /saschazesiger/Free-Proxies/master/proxies/socks5.txt (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0xafe58dc0>: Failed to establish a new connection: [Errno 101] Network is unreachable'))
HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /jetkai/proxy-list/main/online-proxies/txt/proxies-http.txt (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0xafe58940>: Failed to establish a new connection: [Errno 101] Network is unreachable'))
HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /B4RC0DE-TM/proxy-list/main/SOCKS5.txt (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7cbee22050>, 'Connection to raw.githubusercontent.com timed out. (connect timeout=15)'))
HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /B4RC0DE-TM/proxy-list/main/SOCKS4.txt (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7cc015f1d0>, 'Connection to raw.githubusercontent.com timed out. (connect timeout=15)'))
HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /almroot/proxylist/master/list.txt (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7cc62c8f10>, 'Connection to raw.githubusercontent.com timed out. (connect timeout=15)'))
HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /hanwayTech/free-proxy-list/main/socks4.txt (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7cc629ce10>, 'Connection to raw.githubusercontent.com timed out. (connect timeout=15)'))
HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /hanwayTech/free-proxy-list/main/socks5.txt (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7cc629c290>, 'Connection to raw.githubusercontent.com timed out. (connect timeout=15)'))
HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /aslisk/proxyhttps/main/https.txt (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7cbee23810>, 'Connection to raw.githubusercontent.com timed out. (connect timeout=15)'))
HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /HyperBeats/proxy-list/main/socks4.txt (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7cbeb33a50>, 'Connection to raw.githubusercontent.com timed out. (connect timeout=15)'))
HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /hookzof/socks5_list/master/proxy.txt (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7cbeea5910>, 'Connection to raw.githubusercontent.com timed out. (connect timeout=15)'))
HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /B4RC0DE-TM/proxy-list/main/HTTP.txt (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7cc629efd0>, 'Connection to raw.githubusercontent.com timed out. (connect timeout=15)'))
HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /jetkai/proxy-list/main/online-proxies/txt/proxies-socks4.txt (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7cbed5b390>, 'Connection to raw.githubusercontent.com timed out. (connect timeout=15)'))
HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /HyperBeats/proxy-list/main/socks5.txt (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7cbed6c590>, 'Connection to raw.githubusercontent.com timed out. (connect timeout=15)'))
HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /clarketm/proxy-list/master/proxy-list-raw.txt (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7cbed70c90>, 'Connection to raw.githubusercontent.com timed out. (connect timeout=15)'))
HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /jetkai/proxy-list/main/online-proxies/txt/proxies-socks5.txt (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7cbe9e4190>, 'Connection to raw.githubusercontent.com timed out. (connect timeout=15)'))
HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /mmpx12/proxy-list/master/socks4.txt (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7cbe9cbe10>, 'Connection to raw.githubusercontent.com timed out. (connect timeout=15)'))
HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /hanwayTech/free-proxy-list/main/http.txt (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7cbed71350>, 'Connection to raw.githubusercontent.com timed out. (connect timeout=15)'))
HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /roosterkid/openproxylist/main/SOCKS4_RAW.txt (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7cbead3d10>, 'Connection to raw.githubusercontent.com timed out. (connect timeout=15)'))
HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /manuGMG/proxy-365/main/SOCKS5.txt (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7cbe8cdcd0>, 'Connection to raw.githubusercontent.com timed out. (connect timeout=15)'))
HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /hanwayTech/free-proxy-list/main/https.txt (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7cbe9e5d50>, 'Connection to raw.githubusercontent.com timed out. (connect timeout=15)'))
HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /saschazesiger/Free-Proxies/master/proxies/socks4.txt (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7cbecbe1d0>, 'Connection to raw.githubusercontent.com timed out. (connect timeout=15)'))
HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /mmpx12/proxy-list/master/socks5.txt (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7cbecbdb50>, 'Connection to raw.githubusercontent.com timed out. (connect timeout=15)'))
HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /hendrikbgr/Free-Proxy-Repo/master/proxy_list.txt (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7cbecbf610>, 'Connection to raw.githubusercontent.com timed out. (connect timeout=15)'))
HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /roosterkid/openproxylist/main/SOCKS5_RAW.txt (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7cbe7fadd0>, 'Connection to raw.githubusercontent.com timed out. (connect timeout=15)'))
HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /HyperBeats/proxy-list/main/http.txt (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7cbec8c690>, 'Connection to raw.githubusercontent.com timed out. (connect timeout=15)'))
HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /ShiftyTR/Proxy-List/master/socks4.txt (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7cbe5b99d0>, 'Connection to raw.githubusercontent.com timed out. (connect timeout=15)'))
HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /TheSpeedX/PROXY-List/master/socks4.txt (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7cbe333d10>, 'Connection to raw.githubusercontent.com timed out. (connect timeout=15)'))
HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /jetkai/proxy-list/main/online-proxies/txt/proxies-http.txt (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7cbe331d50>, 'Connection to raw.githubusercontent.com timed out. (connect timeout=15)'))
HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /B4RC0DE-TM/proxy-list/main/SOCKS4.txt (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x6f3b0c5610>, 'Connection to raw.githubusercontent.com timed out. (connect timeout=15)'))
HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /B4RC0DE-TM/proxy-list/main/SOCKS5.txt (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x6f3af36610>, 'Connection to raw.githubusercontent.com timed out. (connect timeout=15)'))
HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /almroot/proxylist/master/list.txt (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x6f3b0c66d0>, 'Connection to raw.githubusercontent.com timed out. (connect timeout=15)'))
HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /hanwayTech/free-proxy-list/main/socks4.txt (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x6f41319790>, 'Connection to raw.githubusercontent.com timed out. (connect timeout=15)'))
HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /hanwayTech/free-proxy-list/main/socks5.txt (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x6f3b836f10>, 'Connection to raw.githubusercontent.com timed out. (connect timeout=15)'))
HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /aslisk/proxyhttps/main/https.txt (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x6f3af37550>, 'Connection to raw.githubusercontent.com timed out. (connect timeout=15)'))
HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /HyperBeats/proxy-list/main/socks4.txt (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x6f412b22d0>, 'Connection to raw.githubusercontent.com timed out. (connect timeout=15)'))
HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /hookzof/socks5_list/master/proxy.txt (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x6f3b82e7d0>, 'Connection to raw.githubusercontent.com timed out. (connect timeout=15)'))
HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /B4RC0DE-TM/proxy-list/main/HTTP.txt (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x6f3af36150>, 'Connection to raw.githubusercontent.com timed out. (connect timeout=15)'))
HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /jetkai/proxy-list/main/online-proxies/txt/proxies-socks4.txt (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x6f3b0252d0>, 'Connection to raw.githubusercontent.com timed out. (connect timeout=15)'))
HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /HyperBeats/proxy-list/main/socks5.txt (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x6f3afef050>, 'Connection to raw.githubusercontent.com timed out. (connect timeout=15)'))
HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /clarketm/proxy-list/master/proxy-list-raw.txt (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x6f3afe42d0>, 'Connection to raw.githubusercontent.com timed out. (connect timeout=15)'))
HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /mmpx12/proxy-list/master/socks4.txt (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x6f3b8cefd0>, 'Connection to raw.githubusercontent.com timed out. (connect timeout=15)'))
HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /jetkai/proxy-list/main/online-proxies/txt/proxies-socks5.txt (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x6f3b8c2310>, 'Connection to raw.githubusercontent.com timed out. (connect timeout=15)'))
HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /hanwayTech/free-proxy-list/main/http.txt (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x6f3b16cc50>, 'Connection to raw.githubusercontent.com timed out. (connect timeout=15)'))
HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /roosterkid/openproxylist/main/SOCKS4_RAW.txt (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x6f3b164ad0>, 'Connection to raw.githubusercontent.com timed out. (connect timeout=15)'))
HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /manuGMG/proxy-365/main/SOCKS5.txt (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x6f39c2e010>, 'Connection to raw.githubusercontent.com timed out. (connect timeout=15)'))
HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /hanwayTech/free-proxy-list/main/https.txt (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x6f3afede90>, 'Connection to raw.githubusercontent.com timed out. (connect timeout=15)'))
HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /saschazesiger/Free-Proxies/master/proxies/socks4.txt (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x6f3995a290>, 'Connection to raw.githubusercontent.com timed out. (connect timeout=15)'))
HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /mmpx12/proxy-list/master/socks5.txt (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x6f3b179d50>, 'Connection to raw.githubusercontent.com timed out. (connect timeout=15)'))
HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /hendrikbgr/Free-Proxy-Repo/master/proxy_list.txt (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x6f3b007c10>, 'Connection to raw.githubusercontent.com timed out. (connect timeout=15)'))
HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /ShiftyTR/Proxy-List/master/socks4.txt (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x6f399578d0>, 'Connection to raw.githubusercontent.com timed out. (connect timeout=15)'))
HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /roosterkid/openproxylist/main/SOCKS5_RAW.txt (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x6f4132bd10>, 'Connection to raw.githubusercontent.com timed out. (connect timeout=15)'))
HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /HyperBeats/proxy-list/main/http.txt (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x6f3b007450>, 'Connection to raw.githubusercontent.com timed out. (connect timeout=15)'))
HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /TheSpeedX/PROXY-List/master/socks4.txt (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x6f39961cd0>, 'Connection to raw.githubusercontent.com timed out. (connect timeout=15)'))
HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /saschazesiger/Free-Proxies/master/proxies/socks5.txt (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x6f39a2e150>, 'Connection to raw.githubusercontent.com timed out. (connect timeout=15)'))
HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /jetkai/proxy-list/main/online-proxies/txt/proxies-http.txt (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x6f39a23950>, 'Connection to raw.githubusercontent.com timed out. (connect timeout=15)'))
HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /Zaeem20/FREE_PROXY_LIST/master/socks4.txt (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x6f39a2d390>, 'Connection to raw.githubusercontent.com timed out. (connect timeout=15)'))
HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /ShiftyTR/Proxy-List/master/socks5.txt (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x6f3995d910>, 'Connection to raw.githubusercontent.com timed out. (connect timeout=15)'))
HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /jetkai/proxy-list/main/online-proxies/txt/proxies-https.txt (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x6f39a25250>, 'Connection to raw.githubusercontent.com timed out. (connect timeout=15)'))
HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /TheSpeedX/PROXY-List/master/socks5.txt (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x6f3995cd50>, 'Connection to raw.githubusercontent.com timed out. (connect timeout=15)'))
HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /mertguvencli/http-proxy-list/main/proxy-list/data.txt (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x6f3995e490>, 'Connection to raw.githubusercontent.com timed out. (connect timeout=15)'))
HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /Zaeem20/FREE_PROXY_LIST/master/socks5.txt (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x6f3943a4d0>, 'Connection to raw.githubusercontent.com timed out. (connect timeout=15)'))
HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /mmpx12/proxy-list/master/http.txt (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x6f394398d0>, 'Connection to raw.githubusercontent.com timed out. (connect timeout=15)'))
HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /mmpx12/proxy-list/master/https.txt (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x6f39996590>, 'Connection to raw.githubusercontent.com timed out. (connect timeout=15)'))
HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /roosterkid/openproxylist/main/HTTPS_RAW.txt (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x6f397b4cd0>, 'Connection to raw.githubusercontent.com timed out. (connect timeout=15)'))
HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /RX4096/proxy-list/main/online/http.txt (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x6f396eead0>, 'Connection to raw.githubusercontent.com timed out. (connect timeout=15)'))
HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /RX4096/proxy-list/main/online/https.txt (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x6f39da2250>, 'Connection to raw.githubusercontent.com timed out. (connect timeout=15)'))
HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /saisuiu/uiu/main/free.txt (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x6f396f5910>, 'Connection to raw.githubusercontent.com timed out. (connect timeout=15)'))
HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /saschazesiger/Free-Proxies/master/proxies/http.txt (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x6f392a9350>, 'Connection to raw.githubusercontent.com timed out. (connect timeout=15)'))
HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /ShiftyTR/Proxy-List/master/http.txt (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x6f39d93bd0>, 'Connection to raw.githubusercontent.com timed out. (connect timeout=15)'))
HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /ShiftyTR/Proxy-List/master/https.txt (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x6f39473e90>, 'Connection to raw.githubusercontent.com timed out. (connect timeout=15)'))
HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /sunny9577/proxy-scraper/master/proxies.txt (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x6f3939fe50>, 'Connection to raw.githubusercontent.com timed out. (connect timeout=15)'))
HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /TheSpeedX/PROXY-List/master/http.txt (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x6f397eec90>: Failed to establish a new connection: [Errno 103] Software caused connection abort'))
HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /yemixzy/proxy-list/main/proxy-list/data.txt (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x6f39994150>: Failed to establish a new connection: [Errno 101] Network is unreachable'))
HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /Zaeem20/FREE_PROXY_LIST/master/http.txt (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x6f396ec090>: Failed to establish a new connection: [Errno 101] Network is unreachable'))
HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /Zaeem20/FREE_PROXY_LIST/master/https.txt (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x6f39da34d0>: Failed to establish a new connection: [Errno 101] Network is unreachable'))
HTTPSConnectionPool(host='rootjazz.com', port=443): Max retries exceeded with url: /proxies/proxies.txt (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x6f41275590>: Failed to establish a new connection: [Errno 7] No address associated with hostname'))
HTTPSConnectionPool(host='sheesh.rip', port=443): Max retries exceeded with url: /http.txt (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x6f444a8a90>: Failed to establish a new connection: [Errno 7] No address associated with hostname'))
HTTPSConnectionPool(host='spys.me', port=443): Max retries exceeded with url: /proxy.txt (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x6f39da1c10>: Failed to establish a new connection: [Errno 7] No address associated with hostname'))
HTTPSConnectionPool(host='www.freeproxychecker.com', port=443): Max retries exceeded with url: /result/http_proxies.txt (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x6f3b179e90>: Failed to establish a new connection: [Errno 7] No address associated with hostname'))
HTTPSConnectionPool(host='www.proxy-list.download', port=443): Max retries exceeded with url: /api/v1/get?type=http (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x6f396f59d0>: Failed to establish a new connection: [Errno 7] No address associated with hostname'))
HTTPSConnectionPool(host='www.proxy-list.download', port=443): Max retries exceeded with url: /api/v1/get?type=https (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x6f39473290>: Failed to establish a new connection: [Errno 7] No address associated with hostname'))
